OpenRA.Game/Map/Map.cs

```diff
@@ -722,13 +722,16 @@ public void MakeDefaultPlayers()
 				Players.Add(p.Name, p);
 			}
 
-			Players.Add("Creeps", new PlayerReference
+			if (!Players.ContainsKey("Creeps"))
```

Comment:
This looks like a hack.  We should look into identifying and fixing the root problem instead.